test/test_ops.py

```diff
@@ -1000,6 +1002,43 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
 
+class TestMasksToBoxes:
```

Comment:
You're not using any class features so you can likely move the methods to the module namespace.

Reply:
We grouped some tests in this file in classes because they belong to the same operators, but it doesn't mean we should do it with everything. In this case there's only one test, so we don't really need a class. It's not a huge deal though.

>  It would be nice if the file is rewritten fully using pytest, calling fixtures, etc

It's heavily relying on pytest parametrization already :)
What fixtures are you thinking of?

Reply:
Yes, but the tests above E,g TestBoxIou, TestBoxArea do one test in a class. I would prefer code consistency.

fixture should be both the_functions, rather than calling it, pytest way is to use fixture and pass as param.

Reply:
> Yes, but the tests above E,g TestBoxIou, TestBoxArea do one test in a class. I would prefer code consistency.

Fair enough :)

> fixture should be both the_functions, rather than calling it, pytest way is to use fixture and pass as param.

Hm, I'm not sure I agree here. Fixtures are helpful to have a setup / teardown mechanism, and to have automatically cached function calls, which is useful e.g. if you need the same expensive resource across multiple tests in a class, or in a test module. When none of these conditions are met, there's no reason to use a fixture instead of a simple function